classification1.Rmd

```diff
@@ -472,86 +489,155 @@ perim_concav + annotate("path",
 ### More than two explanatory variables 
 
 Although the above description is directed toward two predictor variables, 
-exactly the same K-nearest neighbours algorithm applies when you
+exactly the same $K$-nearest neighbors algorithm applies when you
 have a higher number of predictor variables (i.e., a higher-dimensional
 predictor space).  Each predictor variable may give us new
 information to help create our classifier.  The only difference is the formula
-for the distance between points. In particular, let's say we have $m$ predictor
+for the distance between points. 
+
+Suppose we have $m$ predictor
```

Comment:
the spacing of these few lines is weird. this one sentence shouldn't be its own paragraph